container-images/scripts/build-cli.sh

```diff
@@ -26,6 +26,13 @@ install_ramalama() {
   ln -sf /usr/bin/podman-remote /usr/bin/podman
   if [ -e "/run/ramalama" ]; then
     python3 -m pip install /run/ramalama --prefix=/usr
+  else
```

Comment:
I suspect this is only part of the draft :)

Reply:
Thanks for the review! While testing `ramalama run`, I noticed that it overrides the container’s `Cmd` with something like:

```json
"Cmd": [
    "/usr/libexec/ramalama/ramalama-run-core",
    "--jinja",
    "-c",
    "2048",
    "--temp",
    "0.8",
    "--threads",
    "6",
    "/mnt/models/model.file"
],
```

This means `ramalama` needs to be installed inside the container. In https://github.com/containers/ramalama/pull/1387, the installation step was removed from Docker-based container builds. I’ve looked into the history and restored the previous behavior for Docker to ensure the functionality is the same as Podman.

Reply:
Yes this needs to be installed inside the container, if it's not, it's a bug 😅